scripts/deploy.sh

```diff
@@ -56,12 +56,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # move OSA variables file to AIO location.
```

Comment:
A detailed comment would be helpful here, explaining why there is a need for an additional file and regarding the required ordering.